rDNA/tests/testthat/test-data-management.R

```diff
@@ -0,0 +1,75 @@
+context("Data management")
+
+conn <- dna_connection("sample.dna")
+
+# Not working
+# test_that("add Attribute", {
+#   expect_output({
+     dna_addAttribute(conn)
+#   }, "A new attribute with ID2was added to the database.")
+# })
+
+# test_that("add Document",{
+#   expect_output({
+     dna_addDocument(conn, date = as.POSIXct("2018-12-02 19:32:19 GMT", tz = "GMT"))
+#   }, "A new document with ID 8 was added to the database.")
+# })
+
+test_that("get Attributes", {
+  expect_equal({
+    dna_getAttributes(conn)
+  }, readRDS("../files/dna_getAttributes.RDS"))
+})
+
+# saveRDS(dna_getAttributes(conn), "../files/dna_getAttributes.RDS")
+
+test_that("get Documents", {
+  expect_equal({
+    dna_getDocuments(conn)
+  }, readRDS("../files/dna_getDocuments.RDS"))
+})
+
+# saveRDS(dna_getDocuments(conn), "../files/dna_getDocuments.RDS")
+
+test_that("remove Attribute", {
+  expect_output({
+   dna_removeAttribute(conn, attribute = 2)
+  }, paste0("Simulation mode: no actual changes are made to the database!\n",
+       "Statements removed: 0"))
```

Comment:
When I run this, I get:
```
Simulation mode: no actual changes are made to the database!
Statements removed: 0
Removal of attribute 2: ID was not found.
```
That's because:
```
> dna_getAttributes(conn)$id
[1] 16  7 14 25 11 19 22
```
Perhaps instead: `dna_removeAttribute(conn, attribute = 25)` and then test against the full output?